packages/codemirror-graphql/src/info.ts

```diff
@@ -74,36 +74,51 @@ CodeMirror.registerHelper(
       (kind === 'Field' && step === 0 && typeInfo.fieldDef) ||
       (kind === 'AliasedField' && step === 2 && typeInfo.fieldDef)
     ) {
+      const header = document.createElement('div');
+      header.className = 'CodeMirror-info-header';
+      renderField(header, typeInfo, options);
       const into = document.createElement('div');
-      renderField(into, typeInfo, options);
+      into.appendChild(header);
```

Comment:
I would like for maintainers such as @yoshiakis and @imolorhe to be able to review this change, to make sure that it doesn't clash with other popular `codemirror-graphql` implementations outside of `graphiql`

Reply:
Makes sense! This would be a new major version for `codemirror-graphql`, so it would not break any dependents. Still good to know if this would be blocking others to update to this new version in any way.

@yoshiakis @imolorhe to sum up the changes for easy digestion:
- For fields the type prefix was removed, i.e. `MyType.myField` -> `myField`
- For args, the type and field was removed, i.e. `MyType.myField(myArg: MyArgType)` -> `myArg: MyArgType`
- The DOM structure of the info tooltip changed (for styling reasons):
  - The first section (i.e. the clickable parts like type and field name) are wrapped in an additional `div`
  - The markdown content for deprecation reasons is wrapped in an additional `div`

Reply:
also CC @stonexer @benjie @sgrove  and others who might be interested in this conversation!

Reply:
@imolorhe yes, essentially just by making it a major version change and adding the equivalent conventional-commits style breaking change note in bold in the changeset markdown! Then it will appear in the gh release and in the changelog

Reply:
Yes, this will show up in the changelog! I also made the changeset message more elaborate and added the bullets from my comment above.